desk/lib/summarize.hoon

```diff
@@ -14,6 +14,48 @@
   [(scot %p our) term (scot %da now) spur]
 ::
 ++  summarize-activity
+  ?:  .^(? %gu (scry-path %channels /$))
+    summarize-activity-new-groups
+  summarize-activity-old-groups
```

Comment:
NB: old-groups kind of brings graph-store era groups to mind but I don't know a better name.